server/src/pages/shot/view.js

```diff
@@ -337,57 +336,58 @@ class Body extends React.Component {
       <Localized id="shotPageConfirmDelete" key="error-4"><div id="shotPageConfirmDelete" hidden></div></Localized>
     ];
 
-    const linkTextShort = shot.urlDisplay;
-
-    const timeDiff = <TimeDiff date={shot.createdDate} />;
-
     let favoriteShotButton = null;
     let trashOrFlagButton;
     let editButton;
-    const highlight = this.state.highlightEditButton ? <div className="edit-highlight" onClick={ this.onClickEdit.bind(this) } onMouseOver={ this.onMouseOverHighlight.bind(this) } onMouseOut={ this.onMouseOutHighlight.bind(this) }></div> : null;
 
+    const highlight = this.state.highlightEditButton ? <div className="edit-highlight" onClick={ this.onClickEdit.bind(this) } onMouseOver={ this.onMouseOverHighlight.bind(this) } onMouseOut={ this.onMouseOutHighlight.bind(this) }></div> : null;
     const activeFavClass = this.props.expireTime ? "" : "is-fav";
     const shouldShow = this.props.isFxaAuthenticated ? "" : "hidden";
-    favoriteShotButton = <div className="fav-wrapper"><Localized id="shotPagefavoriteButton"><button
-      className={`button favorite ${shouldShow} ${activeFavClass}`}
-      title="Favorite this shot"
-      onClick={ this.onClickFavorite.bind(this) }
-    /></Localized></div>;
+
+    favoriteShotButton = <Localized id="shotPageFavoriteButton">
+      <button className={`nav-button ${shouldShow} `} onClick={this.onClickFavorite.bind(this)}>
+        <div className={`icon-favorite favorite ${activeFavClass}`} ></div>
```

Comment:
Is it not possible to achieve the style we want by wrapping the button in the div, not vice-versa?